vnext/ReactUWP/Views/DynamicAutomationPeer.cpp

```diff
@@ -129,10 +140,24 @@ winrt::IInspectable DynamicAutomationPeer::GetPatternCore(
 
 bool DynamicAutomationPeer::IsEnabledCore() const {
   bool disabled =
+      HasAccessibilityState(winrt::react::uwp::AccessibilityStates::Disabled) &&
       GetAccessibilityState(winrt::react::uwp::AccessibilityStates::Disabled);
   return !disabled && Super::IsEnabledCore();
 }
 
+winrt::hstring DynamicAutomationPeer::GetItemStatusCore() const {
```

Comment:
should it be GetItemBusyStatus?
I didn't see you target for other states.

Reply:
Unfortunately we don't have a dedicated automation story for Busy. The guidance from https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.automationproperties.itemstatusproperty is to use ItemStatus property. I'm overriding this here rather than overriding the property if it's set.